### DISCUSSION TITLE
ArraySpray Emitter Position

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#299] [Answers & Questions]
is there a way to translate the emitter position rather than the whole particle system ?

**Comment by tastez** [comment]
which particle system? you can translate the emitter with translate op or transform op at the moment

**Comment by wirmachenbunt** [comment]
sorry for the late answer, i was talking about the arrayspray op, check the helppatch https://cables.gl/edit/hY5lAw . when you translate, you translate the whole particle system incl. spawned particles rather than just the emitter
